src/store/modules/accounts/actions.js

```diff
@@ -15,31 +18,46 @@ const { startLoading, endLoading } = createActionHelpers({
  */
 export default {
   /**
-   * Template
+   * get the account details based on an address
+   * @param {Object} state
    * @param {Function} commit
    * @param {Function} dispatch
-   * @param {*} payload
-   * @return {Promise<any>}
+   * @param {String} address
+   * @return {*}
    */
-  template: function ({ commit, dispatch }, payload) {
-    /**
-     * start load state
-     */
-    startLoading(dispatch, 'blocks/template')
-
-    /**
-     * Return New Promise
-     */
-    return new Promise((resolve, reject) => {
-      /**
-       * End Loading State
-       */
-      endLoading(dispatch, 'blocks/template')
-
-      /**
-       * Resolve template
-       */
-      return resolve('template')
-    })
+  async get ({ state, commit, dispatch }, address) {
+    startLoading(dispatch, 'accounts/get')
+
+    const client = await ae
+
+    const [{ balance }, { transactions }] = await Promise.all([
+      await client.api.getAccountBalance(address),
+      await client.api.getAccountTransactions(address)
+    ])
+
+    const account = { address, balance, transactions }
+
+    if (isEqual(state.account, account)) return
+
+    commit('setAccount', account)
+
+    return endLoading(dispatch, 'accounts/get')
+  },
+
+  async name ({ state, commit, dispatch }, address) {
+    startLoading(dispatch, 'accounts/name')
+
+    if (!process.env.NAME_LOOKUP_MIDDLEWARE_URL) return
+    if (!isEmpty(state.name)) if (new Date().getTime() - state.name.ts < 10000) return
```

Comment:
You can use `Date.now()` instead of `new Date().getTime()`

Reply:
Will update.